### PR TITLE
Remove unnecessary usings part 4

### PR DESCRIPTION
**Pull request by xtqqczze**
`src\Microsoft.PowerShell.Commands.Utility\Microsoft.PowerShell.Commands.Utility.csproj`

_Contributes to #13504._

https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0005

### CODE REVIEW

**Comment by hoodedfoxtailz** [comment]
Power apk

**Comment by xtqqczze** [comment]
closing/reopening to restart tests

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs [hunk 1]
```diff
@@ -10,10 +10,8 @@
 using System.Management.Automation;
 using System.Management.Automation.Internal;
 using System.Management.Automation.Language;
-using System.Management.Automation.Security;
 using System.Reflection;
 using System.Runtime.InteropServices;
-using System.Threading;
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs:16]: CIs are failed on this.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs [hunk 2]
```diff
@@ -10,8 +10,14 @@
...
 using System.Management.Automation.Internal;
 using System.Management.Automation.Language;
+#if !UNIX
+using System.Management.Automation.Security;
+#endif
 using System.Reflection;
 using System.Runtime.InteropServices;
+#if !UNIX
+using System.Threading;
+#endif
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs:20]: Please sort order.

Comment (xtqqczze) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs:20]: The directives have been sorted alphabetically.

Do you want `using System.Management.Automation.Security` and `using System.Threading` within the same `#if !UNIX`?

Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs:20]: Sorry, I skipped something...

**Review by iSazonov — APPROVED** [review]

**Pull request merged**